### PR TITLE
feat: Create 2022-07-25-una-stagione-di-incontri-con-la-community

### PR DESCRIPTION
**Pull request by Francescopog**
@bfabio

## Description

This PR tackles:

* ...
* ...
* ...

In particular, the ...

## Checklist

* [ ] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] Ready for review! :rocket:

## Fixes

Fixes #

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>There is a problem with the commit message
> 2022-07-25-una-stagione-di-incontri-con-la-community.md

@bfabio
- subject may not be empty
- type may not be empty</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 35ea53bf57189ae6cffc932fd0fa5e36b59b2e89
</p>

File: _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+subtitle: "Community call e community lab dedicate ai servizi pubblici digitali: facciamo un bilancio insieme."
+date: 2022-07-25 08:00:00 +0200
+tags: developers designers communitycall communitylab
+categories: news
+author: Francesco Pognante
+image: /assets/icons/logo-it.png
+locale: it
+---
+
+Altri mille di questi partecipanti! Si, perchè sono stati oltre mille le persone che hanno partecipato alla ricca stagione di eventi e occasioni di confronto organizzate da [Developers Italia](https://developers.italia.it/) e [Designers Italia](https://designers.italia.it/). In nove puntate abbiamo discusso assieme di design system del Paese, di sviluppo aperto per i servizi pubblici, eGovernment Benchmark, ma anche di strumenti e risorse pratiche, come Git, Bootstrap Italia 2.0, UI Kit, progettazione di API REST e l’uso container.
```
Comment (bfabio) [inline _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md:13]:
```suggestion
Altri mille di questi partecipanti! Si, perché sono state oltre mille le persone che hanno partecipato alla ricca stagione di eventi e occasioni di confronto organizzate da [Developers Italia](https://developers.italia.it/) e [Designers Italia](https://designers.italia.it/). In nove puntate abbiamo discusso assieme di design system del Paese, di sviluppo aperto per i servizi pubblici, eGovernment Benchmark, ma anche di strumenti e risorse pratiche, come Git, Bootstrap Italia 2.0, UI Kit, progettazione di API REST e l’uso container.
```

File: _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Durante queste occasioni abbiamo condiviso conoscenze, soluzioni e strumenti per migliorare i servizi pubblici digitali con i membri della community, favorendo l’incontro tra tecnici, designer, fornitori e funzionari pubblici. In sostanza, tra tutti coloro che in qualche modo partecipano alle creazione e al mantenimento di servizi pubblici digitali.
+
+Tutto questo non sarebbe stato possibile senza la community di oltre 20.000 persone presenti sui canali [Forum Italia](https://forum.italia.it/) e [Slack Developers](https://slack.developers.italia.it/) composta da quanti si occupano di sviluppo, design, project management nelle PA, in house e aziende fornitrici . Un ringraziamento va quindi dedicato a loro, a chi ha partecipato agli eventi, ma anche a chi è intervenuto con un commento o un messaggio su Slack, a chi ha inviato una pull request su Github e a chi ha in qualche modo contribuito a far crescere a mantenere attiva questa comunità.
```
Comment (bfabio) [inline _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md:17]:
```suggestion
Tutto questo non sarebbe stato possibile senza la community di oltre 20.000 persone presenti sui canali [Forum Italia](https://forum.italia.it/) e [Slack Developers](https://slack.developers.italia.it/) composta da quanti si occupano di sviluppo, design, project management nelle PA, in house e aziende fornitrici. Un ringraziamento va quindi dedicato a loro, a chi ha partecipato agli eventi, ma anche a chi è intervenuto con un commento o un messaggio su Slack, a chi ha inviato una pull request su GitHub e a chi ha in qualche modo contribuito a far crescere a mantenere attiva questa comunità.
```

File: _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+COMMUNITY CALL
+- L’importanza di un design system per i servizi pubblici digitali ([9 febbraio](https://developers.italia.it/it/news/2022/02/22/come-si-realizza-design-servizi-pubblici))
+- eGovernment Benchmark: dall’Europa, i criteri minimi di qualità dei servizi pubblici digitali ([9 marzo](https://developers.italia.it/it/news/2022/03/18/come-eu-valuta-servizi-pubblici-digitali-egovernment-benchmark))
+- Design: la sinergia possibile tra Università e Pubbliche amministrazioni ([6 aprile](https://developers.italia.it/it/news/2022/04/13/pa-universita-progettare-servizi-pubblici-insieme))
+- Esperienze di sviluppo aperto per i servizi pubblici ([18 maggio](https://developers.italia.it/it/news/2022/05/20/sviluppo-open-source-migliorare-servizi-pubblici))
+
+COMMUNITY LAB
+
+- L’uso di Git ([23 febbraio](https://www.youtube.com/watch?v=2Ph4VSyi9lU&list=PLLZQsns4g-N5Eu8xv0L-iYhGU-IWBQSTL&index=9&t=2s))
```
Comment (bfabio) [inline _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md:32]:
```suggestion
- L’uso di Git ([23 febbraio](https://www.youtube.com/watch?v=2Ph4VSyi9lU&list=PLLZQsns4g-N5Eu8xv0L-iYhGU-IWBQSTL&index=9))
```

File: _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+Tutto questo non sarebbe stato possibile senza la community di oltre 20.000 persone presenti sui canali [Forum Italia](https://forum.italia.it/) e [Slack Developers](https://slack.developers.italia.it/) composta da quanti si occupano di sviluppo, design, project management nelle PA, in house e aziende fornitrici . Un ringraziamento va quindi dedicato a loro, a chi ha partecipato agli eventi, ma anche a chi è intervenuto con un commento o un messaggio su Slack, a chi ha inviato una pull request su Github e a chi ha in qualche modo contribuito a far crescere a mantenere attiva questa comunità.
+
+Per rivivere la stagione che ci lasciamo alle spalle e recuperare qualche puntata persa, si possono rivedere i video dei community lab e delle community call sulla [playlist “community”](https://www.youtube.com/playlist?list=PLLZQsns4g-N5Eu8xv0L-iYhGU-IWBQSTL) del canale YouTube del Dipartimento per la trasformazione digitale. 
+
+### Tutti gli eventi
+
+
+COMMUNITY CALL
```
Comment (bfabio) [inline _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md:24]:
```suggestion
## Community call
```

File: _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+- L’importanza di un design system per i servizi pubblici digitali ([9 febbraio](https://developers.italia.it/it/news/2022/02/22/come-si-realizza-design-servizi-pubblici))
+- eGovernment Benchmark: dall’Europa, i criteri minimi di qualità dei servizi pubblici digitali ([9 marzo](https://developers.italia.it/it/news/2022/03/18/come-eu-valuta-servizi-pubblici-digitali-egovernment-benchmark))
+- Design: la sinergia possibile tra Università e Pubbliche amministrazioni ([6 aprile](https://developers.italia.it/it/news/2022/04/13/pa-universita-progettare-servizi-pubblici-insieme))
+- Esperienze di sviluppo aperto per i servizi pubblici ([18 maggio](https://developers.italia.it/it/news/2022/05/20/sviluppo-open-source-migliorare-servizi-pubblici))
+
+COMMUNITY LAB
```
Comment (bfabio) [inline _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md:30]:
```suggestion
## Community lab
```

File: _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+tags: developers designers communitycall communitylab
+categories: news
+author: Francesco Pognante
+image: /assets/icons/logo-it.png
+locale: it
+---
+
+Altri mille di questi partecipanti! Si, perchè sono stati oltre mille le persone che hanno partecipato alla ricca stagione di eventi e occasioni di confronto organizzate da [Developers Italia](https://developers.italia.it/) e [Designers Italia](https://designers.italia.it/). In nove puntate abbiamo discusso assieme di design system del Paese, di sviluppo aperto per i servizi pubblici, eGovernment Benchmark, ma anche di strumenti e risorse pratiche, come Git, Bootstrap Italia 2.0, UI Kit, progettazione di API REST e l’uso container.
+
+Durante queste occasioni abbiamo condiviso conoscenze, soluzioni e strumenti per migliorare i servizi pubblici digitali con i membri della community, favorendo l’incontro tra tecnici, designer, fornitori e funzionari pubblici. In sostanza, tra tutti coloro che in qualche modo partecipano alle creazione e al mantenimento di servizi pubblici digitali.
```
Comment (bfabio) [inline _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md:15]:
```suggestion
Durante queste occasioni abbiamo condiviso conoscenze, soluzioni e strumenti per migliorare i servizi pubblici digitali con i membri della community, favorendo l’incontro tra tecnici, designer, fornitori e funzionari pubblici. In sostanza, tra tutti coloro che in qualche modo partecipano alla creazione e al mantenimento di servizi pubblici digitali.
```

**Review by bfabio — CHANGES_REQUESTED** [review]

File: _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+Tutto questo non sarebbe stato possibile senza la community di oltre 20.000 persone presenti sui canali [Forum Italia](https://forum.italia.it/) e [Slack Developers](https://slack.developers.italia.it/) composta da quanti si occupano di sviluppo, design, project management nelle PA, in house e aziende fornitrici. Un ringraziamento va quindi dedicato a loro, a chi ha partecipato agli eventi, ma anche a chi è intervenuto con un commento o un messaggio su Slack, a chi ha inviato una pull request su GitHub e a chi ha in qualche modo contribuito a far crescere a mantenere attiva questa comunità.
+
+Per rivivere la stagione che ci lasciamo alle spalle e recuperare qualche puntata persa, si possono rivedere i video dei community lab e delle community call sulla [playlist “community”](https://www.youtube.com/playlist?list=PLLZQsns4g-N5Eu8xv0L-iYhGU-IWBQSTL) del canale YouTube del Dipartimento per la trasformazione digitale. 
+
+### Tutti gli eventi
+
+
+## Community call
```
Comment (bfabio) [inline _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md:24]:
```suggestion
#### Community call
```

File: _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+### Tutti gli eventi
+
+
+#### Community call
+- L’importanza di un design system per i servizi pubblici digitali ([9 febbraio](https://developers.italia.it/it/news/2022/02/22/come-si-realizza-design-servizi-pubblici))
+- eGovernment Benchmark: dall’Europa, i criteri minimi di qualità dei servizi pubblici digitali ([9 marzo](https://developers.italia.it/it/news/2022/03/18/come-eu-valuta-servizi-pubblici-digitali-egovernment-benchmark))
+- Design: la sinergia possibile tra Università e Pubbliche amministrazioni ([6 aprile](https://developers.italia.it/it/news/2022/04/13/pa-universita-progettare-servizi-pubblici-insieme))
+- Esperienze di sviluppo aperto per i servizi pubblici ([18 maggio](https://developers.italia.it/it/news/2022/05/20/sviluppo-open-source-migliorare-servizi-pubblici))
+
+## Community lab
```
Comment (bfabio) [inline _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md:30]:
```suggestion
#### Community lab
```

**Review by bfabio — APPROVED** [review]

**Pull request merged**